src/main/webapp/scripts/comments.js

```diff
@@ -0,0 +1,25 @@
+export { makeApiCall, buildRequestBody };
+
+function makeApiCall(videoId, commentText, successfulApiCallback, failedApiCallback) {
```

Comment:
Question: Why use callbacks? Why not return the promise and let the caller do whatever they want to do with it?

```
    return gapi.client.request({
        'path': '/youtube/v3/commentThreads?part=snippet',
        'method': 'POST',
        'body': buildRequestBody(videoId, commentText),
    })
        .then(res => res.json)
```

Reply:
To get the exact error message, the consumer has to go to `err.result.error.message`. I wanted to encapsulate this structure within `postVideoComment()` so that view.js don't have to worry about the structure of error message. That was one motivation I chose callback over promise.  Now I managed to use promise and return the exact error message :D